compiler/rustc_abi/src/lib.rs

```diff
@@ -270,8 +274,10 @@ impl TargetDataLayout {
                     dl.instruction_address_space = parse_address_space(&p[1..], "P")?
                 }
                 ["a", ref a @ ..] => dl.aggregate_align = align(a, "a")?,
+                ["f16", ref a @ ..] => dl.f32_align = align(a, "f16")?,
```

Comment:
```suggestion
                ["f16", ref a @ ..] => dl.f16_align = align(a, "f16")?,
```

Reply:
Thanks, got these two in [f61dbd6](https://github.com/rust-lang/rust/pull/114607/commits/f61dbd64d8eb091a9457c45cc1bca4dfc1f9d5fc)